src/Fixer/ClassNotation/OrderedInterfacesFixer.php

```diff
@@ -153,7 +166,11 @@ protected function applyFix(\SplFileInfo $file, Tokens $tokens): void
             usort($interfaces, function (array $first, array $second): int {
                 $score = self::ORDER_LENGTH === $this->configuration[self::OPTION_ORDER]
                     ? \strlen($first['normalized']) - \strlen($second['normalized'])
-                    : strcasecmp($first['normalized'], $second['normalized']);
+                    : (
+                        (bool) $this->configuration['case_sensitive']
```

Comment:
Is it will be ok to ignore this error in `dev-tools/phpstan/baseline.php`?
<img width="1440" alt="Снимок экрана 2023-08-21 в 12 31 41" src="https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/assets/6824784/557c2eec-1f5b-46e6-9dee-7bc8b0cd7533">

Reply:
You can fix it by re-defining `protected $configuration` with proper array shape that would declare types for each available fields. [Take a look here](https://github.com/phpstan/phpstan/discussions/9686#discussioncomment-6563888) as a reference. But for now it could go to baseline too, especially if other similar errors are dumped there (I don't remember, AFK).

Reply:
Unfortunately, I have never written extensions for PHPSten. I added to the errors to ignore.

Reply:
I did not want you to write PHPStan extension, maybe I should point to [this comment](https://github.com/phpstan/phpstan/discussions/9686#discussioncomment-6549200), not whole discussion 🙂. Anyway, acceptable for me 👍.